examples/bpk-component-modal/stories.tsx

```diff
@@ -59,3 +59,24 @@ export const NoPadding = NoPaddingExample;
 
 export const WithAccessoryView = WithAccessoryViewExample;
 export const Contrast = ContrastExample;
+export const VisualTestDefault = () => Default(true);
+export const VisualTestDefaultWithZoom = {
+  render: () => Default(true),
```

Comment:
I think we can do this: 
```suggestion
  render: () => <><Default isOpen/><Contrast isOpen/><WithAccessoryView isOpen/>
```

Slims down the amount of snaps we need. 

You'll have to turn the examples to read props: 

```
const NoPaddingExample = ({isOpen}) => (
  <ModalContainer title="Modal title" closeLabel="Close modal" padded={false} isOpen={isOpen}>
    This is a default modal. You can put anything you want in here.
  </ModalContainer>
);
```

Reply:
I was curious how this might appear on mobile view perhaps they are just one after the other and not stacked?